File: Rust/WhatBetween/MySolution.rs
fn between(a: i16, b: i16) -> Vec<i16> {
    let mut result: Vec<i16> = Vec::new();
    for n in a..b+1 {
        result.push(n);
    }
    result
}

// Original kata: https://www.codewars.com/kata/55ecd718f46fba02e5000029
// My solution: https://www.codewars.com/kata/reviews/6308763adc82f30001500977/groups/682441a7b08c21a52810f4a1

File: CPP/NeedleInHaystack/MySolution.cpp
#include <string>

using namespace std; 

string sliceString(string str) {
  str.erase(0, 1);
  str.erase(str.end() - 1);
  return str;
}

// Original kata: https://www.codewars.com/kata/56bc28ad5bdaeb48760009b0
// My solution: https://www.codewars.com/kata/reviews/5ab23728f6d55bcd0c000e93/groups/5c9bf5b3ea459e00013a7ab0